docs/output.md

```diff
@@ -315,6 +315,12 @@ When the model calls other tools in parallel with an output tool, you can contro
 
 The `'exhaustive'` strategy is useful when tools have important side effects (like logging, sending notifications, or updating metrics) that should always execute.
 
+!!! warning "Streaming vs Sync Behavior Difference"
+    `run_stream()` behaves differently from `run()` and `run_sync()` when choosing the final result:
+
+    - **`run_stream()`**: The first called tool that **can** produce a final result (output or deferred) becomes the final result
+    - **`run()` / `run_sync()`**: The first **output** tool becomes the final result. If none are called, all **deferred** tools become the final result as `DeferredToolRequests`
```

Comment:
> If none are called

The wording is like this, since we get an `UnexpectedModelBehavior` if both output and deferred tools are called, but none output tools are validated